model/artifact/artifact_file.go

```diff
@@ -147,3 +150,18 @@ func RotateSecrets(toReplace, replacement string, dryRun bool) (map[TaskIDAndExe
 	}
 	return changes, catcher.Resolve()
 }
+
+func EscapeFiles(files []File) []File {
+	var escapedFiles []File
+	for _, file := range files {
+		file.Link = EscapeFile(file.Link)
+		escapedFiles = append(escapedFiles, file)
+	}
+	return escapedFiles
+}
+
+func EscapeFile(path string) string {
```

Comment:
can this be a private function if we only use it for EscapeFiles

Reply:
good call. fixed.